code/src/pages/MovieList.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+
+import { MovieCard } from '../components/MovieCard'
+
+export const MovieList = () => {
+  const [pageNumber, setPageNumber] = useState(1)
+  const MOVIES_URL = `https://api.themoviedb.org/3/movie/popular?api_key=0db2a5ac0c510ce6221cadf605f4d3e6&language=en-US&page=${pageNumber}`
+  const [movies, setMovies] = useState([])
+
+  // This one doesn't really work as I intended, but I can't figure out why. You need to press "Next" twice before it actually changes that page.
+  const changePage = (operator) => {
```

Comment:
Great job with this custom logic, but I think you could achieve same result with `useHistory()` hook from `react-router-dom` package. Check it out!
https://reactrouter.com/web/api/Hooks/usehistory